### PR TITLE
#346 - Update in the browser readme section

### LINKED ISSUE #346: Update home page doco to explain how easy bookmarklets are
Please explain "bookmarklets" better on the home page intro
- I had never heard of them, so ignored them at first
- They are _very_ easy to use, and run after the Javascript. Or after other interactions like clicking on things), which is exactly what I needed.

I was about to raise a Bootstrap Issue, found a similar one that had been closed due to 'you didnt lint first'. So thought I should check first. Blow me if I didn't find some (poorly documented) things that fixed it.

Something like.

You can run bootlint _very_ easily from a web browser bookmark (or "bookmarklet"). 
- Just create a bookmark (of any page). 
- Edit the bookmark, and give this (....) as the URL. 
- Then just click on it when you need it
- View the debug output in the developer console
- e.g. on Chrome: Options | More Tools | Developer Console | "Console" tab) 

Best of all, you can run it at any time, on any page, by just clicking the bookmark , e.g. 
- after the Javascript has run
- after AJAX, clicking things, or other page interactions.

### PR DESCRIPTION
**Pull request by tclindner**
Resolves #346
- Discuss benefits
- Add how to install the bookmarklet
- Add how to use the bookmarklet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,15 +53,35 @@ EOF
...
+1. Create a new bookmark in your browser
+2. Set the name/title equal to something that is easy to remember. Ex: Run Bootlint
+3. Set the URL equal to
 ```js
 javascript:(function(){var s=document.createElement("script");s.onload=function(){bootlint.showLintReportForCurrentDocument([]);};s.src="https://maxcdn.bootstrapcdn.com/bootlint/latest/bootlint.min.js";document.body.appendChild(s)})();
 ```
-Then check the JavaScript console for lint warning messages.
 
-You can also manually download [the browser-ready version of Bootlint](https://github.com/twbs/bootlint/blob/master/dist/browser/bootlint.js).
+Note: The snippet above will ensure you are always running the latest version of bootlint. If you want to reference a specific version of bootlint please see the [BootstrapCDN](http://www.bootstrapcdn.com/bootlint/). Copy the URL and update `s.src=“PASTE-ME-HERE”`.
```
Comment (XhmikosR) [inline README.md]: Can you use https when possible?

File: README.md [hunk 2]
```diff
@@ -53,15 +53,35 @@ EOF
...
+2. A popup will appear informing you if issues were detected
+3. If issues exist, please open the developer tools and select the console tab
+
+#### Alternative Options
+##### Browser ready script
+You can manually download [the browser-ready version of Bootlint](https://github.com/twbs/bootlint/blob/master/dist/browser/bootlint.js).
 
-Bootlint is also available as a unofficial third-party web service at [bootlint.com](http://www.bootlint.com/) which lints your page simply by entering a URL, similar to the [W3C Markup Validation Service](http://validator.w3.org/). **Please note** that we do *not* operate this service and that it may use an outdated version of Bootlint. Therefore it is not the recommended way to use Bootlint.
+##### Third-party service
+Bootlint is available as a unofficial third-party web service at [bootlint.com](http://www.bootlint.com/) which lints your page simply by entering a URL, similar to the [W3C Markup Validation Service](http://validator.w3.org/). **Please note** that we do *not* operate this service and that it may use an outdated version of Bootlint. Therefore it is not the recommended way to use Bootlint.
```
Comment (XhmikosR) [inline README.md]: Same here.

File: README.md (hunk 1 above)
Comment (tclindner) [inline README.md]: Good call. Updated!

File: README.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
 # Bootlint
-
```
Comment (XhmikosR) [inline README.md]: Unrelated change.

File: README.md [hunk 4]
```diff
@@ -215,7 +232,8 @@ See the [GitHub Releases page](https://github.com/twbs/bootlint/releases) for de
 * 2014-10-16 - v0.5.0: Adds several new features. Adds official bookmarklet. Disables auto-lint-on-load in browser. Tweaks some checks. **Not backward compatible**
 * 2014-10-07 - v0.4.0: Adds checks for correct Glyphicon usage and correct modal DOM structure; fixes `.panel-footer` false positive
 * 2014-09-26 - v0.3.0: Several bug fixes and enhancements. **Not backward compatible**
-* 2014-09-23 - v0.2.0: First formal release. Announcement: <http://blog.getbootstrap.com/2014/09/23/bootlint/>
+* 2014-09-23 - v0.2.0: First formal release. Announcement: https://blog.getbootstrap.com/2014/09/23/bootlint/
```
Comment (XhmikosR) [inline README.md]: No need to change this one.

File: README.md [hunk 5]
```diff
@@ -8,7 +7,7 @@
 [![Dependency Status](https://img.shields.io/david/twbs/bootlint.svg)](https://david-dm.org/twbs/bootlint)
 [![devDependency Status](https://img.shields.io/david/dev/twbs/bootlint.svg)](https://david-dm.org/twbs/bootlint#info=devDependencies)
 
-An HTML [linter](https://en.wikipedia.org/wiki/Lint_%28software%29) for [Bootstrap](http://getbootstrap.com) projects
+An HTML [linter](https://en.wikipedia.org/wiki/Lint_%28software%29) for [Bootstrap](https://getbootstrap.com) projects
```
Comment (XhmikosR) [inline README.md]: No need to change this one either.

File: README.md (hunk 3 above)
Comment (tclindner) [inline README.md]: Just a little formatting. I think this is fine.

File: README.md (hunk 4 above)
Comment (tclindner) [inline README.md]: I figured while I was updating the other two urls I would update both of these urls. Is there any reason not to change them?

File: README.md (hunk 3 above)
Comment (XhmikosR) [inline README.md]: No, it's not fine; it's actually wrong. There should always be an empty line after a header.

File: README.md (hunk 4 above)
Comment (XhmikosR) [inline README.md]: Because, we might use CloudFlare, but in general we don't use the https URLs for our links. Also, the angle brackets aren't cosmetic and should be there.

Comment (tclindner) [inline README.md]: Okay, I'll revert them

**Comment by tclindner** [comment]
Hi @cvrebert are you comfortable merging this PR? Thank you!

**Comment by tclindner** [comment]
Hi @cvrebert would you be okay merging this PR in? Thank you!

**Comment by tclindner** [comment]
Hi @XhmikosR are you comfortable merging this PR in? Thank you!!

**Comment by tclindner** [comment]
Sure thing, @XhmikosR! Let me know if the last commit meets your expectations.

**Comment by XhmikosR** [comment]
@tclindner: please fetch upstream master and rebase your branch. Also add these

https://github.com/tclindner/bootlint/commit/b352c0cef43378f0f31669824f60cdcb29907dc1

**Comment by XhmikosR** [comment]
This clearly has gone bad. I'd say delete your branch and do the readme changes from sratch.
